### PR TITLE
Setup backoffice instamint

### PR DESCRIPTION
**Pull request by zrichard22**
If there's anything missing from the setup, let me know

### CODE REVIEW

File: admin/jest.config.ts
Comment (HugoTourneur) [inline admin/jest.config.ts:1]: There is a lot of comments here, should we keep them ?

File: admin/src/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
```
Comment (HugoTourneur) [inline admin/src/app/layout.tsx:18]: Does the lang="en" make sense if the app is multilingual ?

File: admin/jest.config.ts
Comment (zrichard22) [reply admin/jest.config.ts:1]: This is from the default setup of Jest

File: admin/src/app/layout.tsx (hunk 1 above)
Comment (zrichard22) [reply admin/src/app/layout.tsx:18]: 1. This is from the default setup of next.js
2. The admin panel must be in English for me

**Review by HugoTourneur — APPROVED** [review]

**Pull request merged**